digitalocean/Image.py

```diff
@@ -17,15 +17,17 @@ def __init__(self, *args, **kwargs):
         super(Image, self).__init__(*args, **kwargs)
 
     @classmethod
-    def get_object(cls, api_token, image_id_or_slug, use_slug=False):
+    def get_object(cls, api_token, image_id_or_slug):
         """
-            Class method that will return an Image object by ID.
+            Class method that will return an Image object by ID or slug.
         """
-        if use_slug:
+        try:
```

Comment:
avoid `try`.
Check if the value of `image_id_or_slug` is a string or a integer!